MAUI/TreeView/expand-collapse.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title: Expand and Collapse in .NET MAUI TreeView Control | Syncfusion
+description: Learn here all about Expand and Collapse support in the Syncfusion .NET MAUI TreeView (SfTreeView) control and more.
+platform: .NET MAUI
+control: SfTreeView
+documentation: ug
+---
+
+# Expand and Collapse in Xamarin TreeView (SfTreeView)
+
+The TreeView allows you to expand and collapse the nodes either by user interaction on the nodes or by programmatically. 
+
+##  Expand Action Target
+
+ Expanding and Collapsing of nodes can be performed either by tapping the expander view or in both expander view and content view by setting the `ExpandActionTarget` property.
+
+{% tabs %}
+{% highlight xaml %}
+
+<syncfusion:SfTreeView x:Name="TreeView" ExpandActionTarget="Node"/>
+
+{% endhighlight %}
+{% highlight c# %}
+
+// Expands by tapping both expander view and content view.
+treeView.ExpandActionTarget = Syncfusion.Maui.TreeView.TreeViewExpandActionTarget.Node;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Auto Expand Mode
+
+By default, the treeview items will be in collapsed state. You can define how the nodes to be expanded while loading the TreeView by using `AutoExpandMode` property.
+
+The `AutoExpandMode` property is only applicable for bound mode. For Unbound mode you need to set `IsExpanded` property to `true` while creating the nodes, to be in expanded state while loading the TreeView.
+
+* None : All items are collapsed when loaded.
+* RootNodesExpanded : Expands only the root item when loaded.
+* AllNodesExpanded : Expands all the items when loaded.
+
+## Programmatic Expand and Collapse
+
+TreeView allows programmatic expand and collapse based on the `TreeViewNode` and level by using following methods.

```

Comment:
TreeView allows programmatic expansion and collapse based on the TreeViewNode and level using the following methods.